cmd/rad/cmd/root.go

```diff
@@ -211,14 +221,38 @@ func initSubCommands() {
 	runCmd, _ := run.NewCommand(framework)
 	RootCmd.AddCommand(runCmd)
 
-	showCmd, _ := resource_show.NewCommand(framework)
-	resourceCmd.AddCommand(showCmd)
+	resourceShowCmd, _ := resource_show.NewCommand(framework)
+	resourceCmd.AddCommand(resourceShowCmd)
 
-	listCmd, _ := resource_list.NewCommand(framework)
-	resourceCmd.AddCommand(listCmd)
+	resourceListCommand, _ := resource_list.NewCommand(framework)
+	resourceCmd.AddCommand(resourceListCommand)
 
-	deleteCmd, _ := resource_delete.NewCommand(framework)
-	resourceCmd.AddCommand(deleteCmd)
+	resourceCreateCmd, _ := resource_create.NewCommand(framework)
+	resourceCmd.AddCommand(resourceCreateCmd)
+
+	resourceDeleteCmd, _ := resource_delete.NewCommand(framework)
+	resourceCmd.AddCommand(resourceDeleteCmd)
+
+	resourceProviderNewCmd, _ := resourceprovider_new.NewCommand(framework)
+	resourceProviderCmd.AddCommand(resourceProviderNewCmd)
+
+	resourceProviderShowCmd, _ := resourceprovider_show.NewCommand(framework)
+	resourceProviderCmd.AddCommand(resourceProviderShowCmd)
+
+	resourceProviderListCmd, _ := resourceprovider_list.NewCommand(framework)
+	resourceProviderCmd.AddCommand(resourceProviderListCmd)
+
+	resourceProviderCreateCmd, _ := resourceprovider_create.NewCommand(framework)
```

Comment:
I wonder what the difference is between the New and the Create... I guess I will soon find out :)